docker/docker-compose-embedded.yaml

```diff
@@ -14,14 +17,25 @@ services:
       - 6002:6002
     depends_on:
       - ipfs
+      - lotus
     environment:
       - TEXPOWERGATE_EMBEDDED=true
+      - TEXPOWERGATE_LOTUSHOST=/dns4/lotus/tcp/7777
```

Comment:
Now, a run of Powergate with _embedded_ flag is _almost_ like a real-case... the Lotus node it's _outside_ and calls the APIs _as usual_. It still keeps making ephemeral Powergate repos for convenience.